CHANGELOG.md

```diff
@@ -5,7 +5,7 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
-## [Unreleased] (beta)
+## [v0.19.0] (beta) - 2024-02-06
```

Comment:
nit: We typically leave `## [Unreleased] (beta)` at the top. You can use this PR as a ref https://github.com/alcionai/corso/pull/4961/files